tests/callbacks/test_device_stats_monitor.py

```diff
@@ -69,6 +77,44 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
             for f in fields:
                 assert any(f in h for h in metrics.keys())
```

Comment:
```suggestion
                assert any(f in h for h in metrics)
```

Reply:
also, shouldn't this be `all(...)`?

Reply:
No. We are looping through all the keys we want to check with `for f in fields:`. We just want to make sure the key appears in the logged metrics.

Ex: `f` could be `cpu_percent` and the metrics could be:
```
metrics = {'on_train_batch_start/cpu_percent': 18.2, 'on_train_batch_start/cpu_swap_percent': 47.0, 'on_train_batch_start/cpu_vm_percent': 66.3}
```

If we did `all()` when `f = "cpu_percent"` that would be checked that all logged metrics were for `cpu_percent`, which is not what we want. We just want to make sure that all metric keys appear in the logged metrics.

Reply:
oh! okay.. I misinterpreted it.